### DISCUSSION TITLE
Delete after transfer

### DISCUSSION THREAD

**Discussion by PhilC813** [canopas/cloud-gallery#49] [Feedback & Suggestions]
I just discovered your project during my quest on building a more sustainable cloud storage photo solution for my family. Your app is basically exactly what I hoped existed. Thank you for making it happen.

I was wondering if, in the future, there could be something similar to PhotoSync's "[delete after transfer](https://www.photosync-app.com/support/ios/answers/how-to-delete-photos-or-videos#:~:text=delete%20successfully%20transferred%20photos%20and%20videos%20after%20the%20transfer)" feature. On iOS, due to the system (or App Store) limitations, a system prompt appears once a transfer is over to ask if the user wants to proceed with the deletion. On Android, it is done without additional user intervention.

This feature has been useful for one of my family members who has photography as a hobby but is not so tech-savvy. He never really understood whether he was making edits to the local or cloud copies of the files, and the sync engines of cloud backup services are rarely designed to detect the changes to a file and replicate it on the other end. As such, PhotoSync allowed him to only have one copy of the files at a time; either in the cloud or locally on the device, but never both, and that made sure edits wouldn't be lost in a cleanup process.

I also personally like to offload media to the cloud and clear the files from my device afterwards in order to save space (assuming that the cloud provider is trustworthy, and Google has proved to be for my last few years of using Google Drive & Google Photos).

I hope you can consider this suggestion. Thank you!

**Comment by cp-pratik-k** [comment]
> I just discovered your project during my quest on building a more sustainable cloud storage photo solution for my family. Your app is basically exactly what I hoped existed. Thank you for making it happen.
> 
> I was wondering if, in the future, there could be something similar to PhotoSync's "[delete after transfer](https://www.photosync-app.com/support/ios/answers/how-to-delete-photos-or-videos#:~:text=delete%20successfully%20transferred%20photos%20and%20videos%20after%20the%20transfer)" feature. On iOS, due to the system (or App Store) limitations, a system prompt appears once a transfer is over to ask if the user wants to proceed with the deletion. On Android, it is done without additional user intervention.
> 
> This feature has been useful for one of my family members who has photography as a hobby but is not so tech-savvy. He never really understood whether he was making edits to the local or cloud copies of the files, and the sync engines of cloud backup services are rarely designed to detect the changes to a file and replicate it on the other end. As such, PhotoSync allowed him to only have one copy of the files at a time; either in the cloud or locally on the device, but never both, and that made sure edits wouldn't be lost in a cleanup process.
> 
> I also personally like to offload media to the cloud and clear the files from my device afterwards in order to save space (assuming that the cloud provider is trustworthy, and Google has proved to be for my last few years of using Google Drive & Google Photos).
> 
> I hope you can consider this suggestion. Thank you!

Thank you for your amazing feedback! We're so glad to hear that our app is exactly what you need. Your suggestion for a "delete after transfer" feature is beneficial, We'll consider this for future updates.

**Reply by PhilC813** [reply]
Thank you! And just to confirm, is this project still alive?

**Reply by cp-pratik-k** [reply]
Yes, currently we are working on cloud-related implementations. We will consider this in future updates.
